lib/api/api_auth.go

```diff
@@ -35,6 +35,7 @@ func emitLoginAttempt(success bool, username, address string, evLogger events.Lo
 	})
 	if !success {
 		l.Infof("Wrong credentials supplied during API authorization from %s", address)
+		antiBruteForceSleep()
```

Comment:
This change is less obvious in the overall diff, but it's clearer in the https://github.com/syncthing/syncthing/pull/9159/commits/e15b862d3819278aebdbd6ddb530deb9825f7ed6 commit that every invocation of `emitLoginAttempt(false, ...)` was accompanied by an `antiBruteForceSleep()` call, and `antiBruteForceSleep()` was called nowhere else.

Reply:
Mmmm not so enthusiastic about this hiding as a side effect in a function that is otherwise just about emitting an event.

Reply:
Ok, reverted that change.